code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/configuration/ConfigurationRulesManager.kt

```diff
@@ -151,7 +151,11 @@ internal class ConfigurationRulesManager(
             return false
         }
 
-        val cacheDir = File(applicationCacheDir, ADOBE_CACHE_DIR)
+        // Represents the directory where bundled rules will be cached for reading and extracting.
+        val cacheDir = File(
+            applicationCacheDir,
+            ADOBE_CACHE_DIR + File.separator + RULES_CACHE_FOLDER + File.separator + BUNDLED_RULES_DIR
```

Comment:
Can you add this as a constant. 
`const val BUNDLED_RULES_PATH = ADOBE_CACHE_DIR + File.separator + RULES_CACHE_FOLDER + File.separator + BUNDLED_RULES_DIR`

Reply:
Discussed that const vals cannot have computations.